pkg/storage/s3/s3.go

```diff
@@ -437,12 +437,22 @@ func (d *driver) RemoveStorage(cr *opapi.Config, modified *bool) error {
 		return err
 	}
 
+	// Wait until the bucket does not exist
```

Comment:
This probably isn't doing a whole lot toward fixing the issue, but I think it's worth having